src/main/java/longah/Transaction.java

```diff
@@ -0,0 +1,72 @@
+package longah;
+
+/**
+ * Represents a transaction between two members.
+ */
+public class Transaction {
+    private Member from;
+    private Member to;
+    private double amount;
+
+    /**
+     * Constructs a new Transaction instance.
+     *
+     * @param from   The member who owes the amount.
+     * @param to     The member who is owed the amount.
+     * @param amount The amount of the transaction.
+     */
+    public Transaction(Member from, Member to, double amount) {
```

Comment:
Is there a way to implement this to allow for multiple payment targets as opposed to creating a transaction object for every payment target, i.e. arraylist for `to` and `amount`? Would be good if could add an attribute for transaction description as well for optional user input.